src/components/LeafletMap.js

```diff
@@ -100,7 +100,8 @@ const mapProperty = {
   minZoom: 2,
   maxZoom: 16,
   zoomControl: true,
-  layers: [worldImageryMapLayer, labelLayer]
+  layers: [worldImageryMapLayer, labelLayer],
+  maxBoundsViscosity: 0.5
```

Comment:
Is this being used?

Reply:
the maxBoundsViscoscity is used to control the bounce strength from 0 to 1. I found the default a very bouncy. so I just set half

Reply:
Cool :taco: